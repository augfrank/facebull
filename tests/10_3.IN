M0   C0   C4  2705
M1   C0   C7  2937
M2   C1   C2  2054
M3   C1   C4  2143
M4   C1   C6  2140
M5   C1   C7  2181
M6   C2   C1  2907
M7   C2   C5  2369
M8   C2   C8  2699
M9   C3   C0  2372
M10   C3   C4  2994
M11   C3   C8  2079
M12   C4   C1  2251
M13   C4   C3  2694
M14   C4   C7  2086
M15   C4   C8  2641
M16   C4   C9  2760
M17   C5   C2  2865
M18   C5   C4  2291
M19   C5   C6  2214
M20   C5   C8  2554
M21   C6   C0  2453
M22   C6   C1  2425
M23   C6   C3  2017
M24   C6   C4  2033
M25   C6   C8  2038
M26   C6   C9  2237
M27   C7   C1  2124
M28   C7   C3  2666
M29   C7   C9  2238
M30   C8   C4  2240
M31   C8   C5  2017
M32   C8   C9  2207
M33   C9   C1  2537
M34   C9   C5  2402
M35   C9   C6  2116
