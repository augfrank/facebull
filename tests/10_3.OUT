22604
0 2 7 9 14 20 23 27 32 35
